### PR TITLE
Update Jira with extra fields and multi-channel support

### PR DESCRIPTION
**Pull request by idoburstein**
Add Jira extra fields, add Jira to multi-channel functionality, fix docs, templates work, add Jira to multi-channel unit test (only new issue, not add comment as we can't do both functionalities at the same time). Solve CCITNOTES-57 and CCITNOTES-58.

### CODE REVIEW

**Comment by waynesun09** [comment]
This fixes:
https://github.com/RedHatQE/notify-service/issues/41

File: app/api/endpoints/message.py [hunk 1]
```diff
@@ -20,9 +19,9 @@ def param_err(err: str) -> Any:
 
 @router.post("/", response_model=schemas.Msg)
 async def msg_multi_tgts(
-    target: str = Query(
-        ...,
-        description="The message targets seperate by ',', e.g. email,gchat,slack,irc,message_bus"
+    target: List[str] = Query(
+        "email", enum=["email", "gchat", "slack", "irc", "message_bus", 'jira'],
```
[the remark this answers is not in the document]
Comment (idoburstein) [reply app/api/endpoints/message.py:23]: Screenshot from 2022-06-14 16-24-58
This is what the SwaggerUI looks like

[the remark this answers is not in the document]
Comment (idoburstein) [reply app/api/endpoints/message.py:23]: I added the ..., will commit and push soon

[the remark this answers is not in the document]
Comment (waynesun09) [reply app/api/endpoints/message.py:23]: Got it, the UI looks good.

File: app/api/endpoints/message.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ def param_err(err: str) -> Any:
 @router.post("/", response_model=schemas.Msg)
 async def msg_multi_tgts(
     target: List[str] = Query(
-        "email", enum=["email", "gchat", "slack", "irc", "message_bus", 'jira'], 
+        ..., enum=["email", "gchat", "slack", "irc", "message_bus", 'jira'],
```
Comment (waynesun09) [inline app/api/endpoints/message.py:23]: better new line with each parameter

**Pull request closed without merging**